### ISSUE TITLE
Missing Access Key details in GuardDuty finding resource

### ISSUE THREAD

**Issue by yatindra** [aws/aws-sdk-java#1426]
AWS Console shows the Access Key details (Access Key, Principal ID, Username) for GuardDuty API Access findings.

This is also documented in the `GetFindings` API Docs found here: https://docs.aws.amazon.com/guardduty/latest/ug/get-findings.html

```
{
    "findings": [
        {
            "resource": {
                "accessKeyDetails": {
                    "accessKeyId": "string",
                    "principalId": "string",
                    "userType": "string",
                    "userName": "string"
                }
            }
        }
    ]
}
```

But the same is not available in the `com.amazonaws.services.guardduty.model.Resource`

**Comment by dagnir** [comment]
@yatindra Thanks for reporting this to us. Will follow up with the service team to see about updating the service models.

**Comment by yatindra** [comment]
@dagnir Any update on this? Really need this.

**Comment by shorea** [comment]
@yatindra apologies for the delay, we have tentatively scheduled this for release tomorrow. I'll keep you posted.

**Comment by shorea** [comment]
@yatindra this is now available in version 1.11.270 of the SDK.
